legal-api/report-templates/template-parts/common/businessDetails.html

```diff
@@ -117,7 +121,11 @@
             </td>
             <td>
                {% if header.status == 'COMPLETED' %}
-               <div>{{business.identifier}}</div>
+                  {% if business.identifier.startswith('T') %}
```

Comment:
Did you verify this? I am sure if identifier will be present in temp mode (not mandatory field in json anyways)
```suggestion
                  {% if not business.identifier or business.identifier.startswith('T') %}
```
Do the same check in registration and amalgamationApplication as well